### ISSUE TITLE
PPR links not working

### ISSUE THREAD

**Issue by ryscher** [CDL-Dryad/dryad-product-roadmap#2515]
See original description and sample URLs in [SF #00022666](https://dryad.lightning.force.com/lightning/r/Case/5003h00000bUQe2AAG/view)

Some PPR links are not working correctly. They bring up the page that says a download is going to happen, but the download never starts.

**Comment by bryanmgee** [comment]
Another instance of this with a small dataset (~100 kB) with no journal metadata ([SF case](https://dryad.lightning.force.com/lightning/r/Case/5003h00000bUgVLAA0/view); [dataset](https://datadryad.org/stash/ds_admin?q=%22Datasets+and+R+code+from%3A+Assessing+drivers+of+dispersal+for+bees+in+fragmented+landscapes+using+population+genomics+and+GIS%22&sort=relevance&commit=Search&tenant=&editor_id=&curation_status=&publication_name=&sponsor_org=&utf8=%E2%9C%93))

**Comment by ryscher** [comment]
@bryanmgee This issue is now fixed.

**Comment by ryscher** [comment]
@sfisher This problem was caused by the DataCite tracker, so I disabled the tracker on these pages. See https://github.com/CDL-Dryad/dryad-app/pull/1196

Can you look into how to re-enable the tracker?

**Comment by bryanmgee** [comment]
@ryscher Interesting observation I just had. I tried all 5 of the ones that are linked in the ticket on my Firefox, and none of them worked. However, I tried the 5 PPR links for the datasets in my Curation queue right now, and they all worked. I don't have any adblocker extensions or anything on Firefox. However, the five that did not work all worked fine in Chrome, in which I also don't have any extensions.
